### PR TITLE
Task4 ready for Review

### PR DESCRIPTION
**Pull request by martinmilev**
[Presentation](https://martinmilev.github.io/htmlPages/task4/index.html)

### CODE REVIEW

File: task4/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<html>
+    <head>
+    	<title>FieldCheck</title>
+    	<style type="text/css">
```
Comment (freekman) [inline task4/index.html]: move the css and the js in sepatrate files

File: task4/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+			#submit {
+				margin: 20px;
+				width: 80px;
+				height: 30px;
+			}
+		</style>
+
+        <script type="text/javascript">
+
+            function validate()
```
Comment (freekman) [inline task4/index.html]: from the looks of the method i dont know what are you validating
 ``` validate(field)``` , ```isEmpty(field)```

File: task4/index.html (hunk 1 above)
Comment (martinmilev) [inline task4/index.html]: fixed

File: task4/index.html (hunk 2 above)
Comment (martinmilev) [inline task4/index.html]: fixed

File: task4/index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <head>
+    	<title>FieldCheck</title>
+        <title>Links</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="validator.js"></script>
+    </head>
+    <body>
+    	<h1>Check if the field is empty.</h1>
+        <form name="field" id="field" method="post">
+            <input type="text" name="text" id="text"></br>
```
Comment (freekman) [inline task4/index.html]: ok add a second input field and check if it is also empty

Comment (martinmilev) [inline task4/index.html]: done

**Comment by freekman** [comment]
update your ui so can check it out

**Comment by martinmilev** [comment]
ui uodated.

File: task4/validator.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+function isEmpty(){
+    if (!field.text1.value){
```
Comment (freekman) [inline task4/validator.js]: if you have 1000 fields how many ifs will you hava

Comment (martinmilev) [inline task4/validator.js]: Trying to write it something like this. But can't make it work for some reason.
```javascript
function isEmpty(field){
	var value = document.getElementsById(field).value;
    if (value.length < 1){
        alert (field.concat(" is empty!"));
    }
}
```

**Comment by freekman** [comment]
LGTM

**Pull request closed without merging**